libs/openchallenges/ui/src/lib/challenge-card/challenge-card.component.html

```diff
@@ -12,12 +12,12 @@
       <p class="card-subtitle" *ngIf="platform">
         {{ platform.name }}
       </p>
-      <p class="card-desc">{{ challenge.headline }}</p>
+      <p class="mat-caption">{{ challenge.headline }}</p>
     </div>
     <div class="card-footer">
       <div class="difficulty-tag mat-small">
-        <mat-icon aria-hidden="true" class="card-icon">timeline</mat-icon>
-        {{ difficulty }}
+        <mat-icon aria-hidden="true" class="card-icon">auto_awesome</mat-icon>
+        Some incentive
```

Comment:
Can you use `{{ challenge.incentives }}` instead? You might need some adjustment since `incentives` is a lowercased array.